controllers/dockermachine_controller.go

```diff
@@ -243,5 +241,48 @@ func (r *DockerMachineReconciler) SetupWithManager(mgr ctrl.Manager) error {
 				ToRequests: util.MachineToInfrastructureMapFunc(infrav1.GroupVersion.WithKind("DockerMachine")),
 			},
 		).
+		Watches(
+			&source.Kind{Type: &infrav1.DockerCluster{}},
+			&handler.EnqueueRequestsFromMapFunc{
+				ToRequests: handler.ToRequestsFunc(r.DockerClusterToDockerMachines),
+			},
+		).
 		Complete(r)
 }
+
+// DockerClusterToDockerMachines is a handler.ToRequestsFunc to be used to enqeue
+// requests for reconciliation of DockerMachines.
+func (r *DockerMachineReconciler) DockerClusterToDockerMachines(o handler.MapObject) []ctrl.Request {
+	result := []ctrl.Request{}
+	c, ok := o.Object.(*infrav1.DockerCluster)
+	if !ok {
+		r.Log.Error(errors.Errorf("expected a DockerCluster but got a %T", o.Object), "failed to get DockerMachine for DockerCluster")
+		return nil
+	}
+	log := r.Log.WithValues("DockerCluster", c.Name, "Namespace", c.Namespace)
+
+	cluster, err := util.GetOwnerCluster(context.Background(), r.Client, c.ObjectMeta)
```

Comment:
I think technically `context.TODO()` is supposed to be used if you need a context but don't have one passed in to this function. From what I recall, `context.Background()` really should only be created in `main()`. (I realize we are in no way consistent about this across the CAP* repos).

Reply:
totally fair. I noticed the CAPA implementation also uses .Background, but I'll update this

Reply:
missed this one

Reply:
fixed it!